apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx

```diff
@@ -112,18 +120,6 @@ export const SwipeUpDown = forwardRef<SwipeUpDownRef, SwipeUpDownProps>(
       };
     });
 
-    const TransactionList = () => (
```

Comment:
On every render of SwipeUpDown, this was creating a new instance of `TransactionList`, forcing a re-render of `TransactionList`, and all its children.